lib/css/CssModulesPlugin.js

```diff
@@ -432,6 +443,55 @@ class CssModulesPlugin {
 					/** @type {CssModule[] | undefined} */
 					const modules = orderedCssModulesPerChunk.get(chunk);
 					if (modules !== undefined) {
+						const modulesExcludeFromURL = [];
+						/** @type {Map<string, {modules: CssModule[], entry?: CssModule}>} */
+						const URLRequestModulesMap = new Map();
+						for (const module of modules) {
```

Comment:
We should avoid it here

Reply:
I want collect all modules from a `new URL` if we want to support `@import` for it

Reply:
I think we need to search another approach... I will try to find time on this, you can keep it as is right now